tests/asr/wav2vec2/test_clean.py

```diff
@@ -0,0 +1,52 @@
+"""Unit tests for the `asr.wav2vec2.clean` module."""
+
+from copy import deepcopy
+
+import pytest
+
+from coral_models.asr.wav2vec2.clean import clean_dataset, clean_transcription
+
+
+def test_clean_dataset(dataset, cfg) -> None:
```

Comment:
Maybe we should store clean versions of the dataset instead to use for testing?

Right now, we basically just compare calling map on the dataset with list comprehension.

Reply:
I would like to test the cleaning process rather than storing a cleaned version, since the cleaning code will probably change over time. But you're right that not much value is gained from `test_dataset` currently. But with the new refactoring, where `clean_transcription` takes in the `punctuation_regex` and `conversion_dict` variables, it makes more sense to test them both.

But, in general, I'd like to have a small dummy dataset used in tests rather than a fully fledged large(ish) dataset like Common Voice.

Reply:
I've now updated the `tests.asr.wav2vec2.test_clean` module, and also added a dummy dataset, consisting of the first 5 samples of the Danish part of Common Voice.